WordPress/Classes/ViewRelated/Domains/Domain registration/DomainPurchasingWebFlowController.swift

```diff
@@ -0,0 +1,338 @@
+import UIKit
+import AutomatticTracks
+import Sentry
+
+final class DomainPurchasingWebFlowController {
+
+    // MARK: - Constants
+
+    fileprivate enum Constants {
+        static let checkoutWebAddress = "https://wordpress.com/checkout/"
+        static let storeSandboxCookieName = "store_sandbox"
+        static let storeSandboxCookieDomain = ".wordpress.com"
+        static let checkoutSuccessURLPrefix = "https://wordpress.com/checkout/thank-you/"
+        static let checkoutURLPrefix = "https://wordpress.com/checkout"
```

Comment:
(Not urgent) It would be a better practice to have a base url constant defined and add the rest as parameters rather than redefining "https://wordpress.com/" multiple times.

Reply:
Resolved in c9a94701f01a4bd7872985b5dce28c139248c68a